app/helper_test.go

```diff
@@ -707,3 +707,15 @@ func (th *TestHelper) AddPermissionToRole(permission string, roleName string) {
 
 	utils.EnableDebugLogForTest()
 }
+
+// NewTestId is used for testing as a replacement for model.NewId(). It is a [A-Z0-9] string 26
+// characters long. It replaces every odd character with a digit.
```

Comment:
Add a comment that this is a copy of the `storetest/NewTestId` function. And also add a comment in the original function that there's another duplicate in app/helper_test.go